tests/drivers/watchdog/wdt_basic_api/testcase.yaml

```diff
@@ -150,3 +151,8 @@ tests:
     filter: dt_compat_enabled("ti,tps382x")
     platform_allow: mimxrt1050_evk/mimxrt1052/hyperflash
     extra_args: DTC_OVERLAY_FILE="boards/mimxrt1050_evk_ti_tps382x.overlay"
+  drivers.watchdog.nxp_sram_noecc:
+    extra_args: DTC_OVERLAY_FILE="${BOARD_DIR}/dts/sram_noecc.overlay"
```

Comment:
I think it's better to do EXTRA_DTC_OVERLAY_FILE= , but will let @hakehuang comment about that. Also, I'm not sure you need to put the whole filepath prefix, i could be remembering wrong or mixing something up , but i think the dts/ folder of the board is added to the searching path for this variable already

Reply:
Thanks for the remind, boards/<board>/dts was already in the search list.